array-methods.js

```diff
@@ -58,20 +90,44 @@ var stateSums = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var sumOfHighInterests = null;
-
+const sumOfHighInterests = Object.keys(stateSums)
+.filter(key => {
+  return !(selectStates.includes(key));
+})
+.reduce((sum, key) => {
+  // if (Math.round((stateSums[key] * 0.189)) > 50000) {
+    return sum + Math.round((stateSums[key] * 0.189));
```

Comment:
I think this is summing all values, Jay made a change to the test so that the new test works only if the values are greater than 50,000. Please pull in the changes Jay made and update this section so the new test passes.